src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -81,6 +85,10 @@ export class ShelleyWallet extends Wallet implements WalletInterface {
     return this.storage.write(`/wallet/${this.id}/data`, this.toJSON())
   }
 
+  clear() {
```

Comment:
maybe a good place to clear the cache of all services. `clearServices`

Reply:
`resync`/`removeWallet` need to call it

Reply:
It is being used when the portfolio is deleted/resynced 👍🏼

https://github.com/Emurgo/yoroi-mobile/pull/2189#discussion_r1004713812